spec/features/visitor_can_view_items_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "View items" do
+  let!(:item_one) {Item.create(name: "Canon 7d",
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.